src/core/ConfigManager.cpp

```diff
@@ -425,6 +441,11 @@ void ConfigManager::loadConfigFile()
 		QDir().mkpath( userProjectsDir() );
 		QDir().mkpath( userSamplesDir() );
 		QDir().mkpath( userPresetsDir() );
+		QDir().mkpath( userGigDir() );
+		QDir().mkpath( userSf2Dir() );
+		QDir().mkpath( vstDir() );
```

Comment:
Noticed the naming convention of this `vstDir()` vs. `usersVstDir()`... also many Windows users historically have the `vstDir()` set to a restricted area (such as `Program Files/Steinberg/`).  Does `mkpath(...)` fail silently if the area can't be written to?